### DISCUSSION TITLE
Ability to choose the region of the Popular feed

### DISCUSSION THREAD

**Discussion by fwgx** [Docile-Alligator/Infinity-For-Reddit#1350] [Not planned Ideas] [answered]
In the reddit website when viewing popular posts you can choose the region in which they are popular, for example I'm in the UK, so I like seeing what's popular here.  Infinity seems to either use the "everywhere" or the "US" option only which makes using the Popular feature in the app a bit pointless for me.

Add an option in the settings or a filter on the Popular page at the top maybe to allow the user to select the region the popular posts are displayed for.

Thanks

**Comment by fwgx** [comment]
This is the filter I'm talking about...
![Screenshot_20230130_205229](https://user-images.githubusercontent.com/7432335/215592588-99e992c8-da3a-4d71-ba89-44eb2fc992a1.png)

**Comment by Wladefant** [comment, marked as answer]
It seems like it is not exposed to the API https://www.reddit.com/r/BoostForReddit/comments/jt7w4g/boost_only_shows_regional_content_in_rpopular_i/
